packages/drivekit_driver_data/flutter_drivekit_driver_data_android/pigeon/messages.dart

```diff
@@ -18,4 +18,1094 @@ import 'package:pigeon/pigeon.dart';
+  @async
+  bool deleteTrip(String itinId);
+}
+
+/// the response returned when gettings trips
+class PigeonGetTripsResponse {
+  /// Creates a PigeonGetTripsResponse instance
+  PigeonGetTripsResponse({required this.status, required this.trips});
+
+  /// trip synchronization status
+  final PigeonTripSyncStatus status;
+
+  /// fetched trips
+  final List<PigeonTrip?> trips;
+}
+
+/// the response returned when gettings one trip
+class PigeonGetTripResponse {
+  /// Creates a PigeonGetTripResponse instance
+  PigeonGetTripResponse({required this.status, required this.trip});
+
+  /// trip synchronization status
+  final PigeonTripSyncStatus status;
+
+  /// fetched trips
+  final PigeonTrip? trip;
+}
+
+/// Trip synchronization status enum
+enum PigeonTripSyncStatus {
+  /// Synchronization has been successfully performed
+  noError,
+
+  /// SynchronizationType has been set to cache.
+  cacheDataOnly,
+
+  /// Synchronization has failed,
+  /// only trips previously synchronized are returned
+  failedToSyncTripsCacheOnly,
+
+  /// Safety Events synchronization failed
+  failedToSyncSafetyEvents,
+
+  /// A synchronization is in progress, only trips previously
+  /// synchronized are returned until the synchronization is finished
+  syncAlreadyInProgress
+}
+
+/// Trip class
+class PigeonTrip {
+  PigeonTrip({
+    required this.itinId,
+    required this.startDate,
+    required this.endDate,
+    required this.departureCity,
+    required this.arrivalCity,
+    required this.departureAddress,
+    required this.arrivalAddress,
+    required this.vehicleId,
+    required this.comments,
+    required this.tripStatistics,
+    required this.ecoDriving,
+    required this.fuelEstimation,
+    required this.safety,
+    required this.advancedEcoDriving,
+    required this.advancedFuelEstimation,
+    required this.advancedSafety,
+    required this.pollutants,
+    required this.tireWear,
+    required this.brakeWear,
+    required this.driverDistraction,
+    required this.itineraryData,
+    required this.logbook,
+    required this.safetyEvents,
+    required this.speedingStatistics,
+    required this.energyEstimation,
+    required this.advancedEnergyEstimation,
+    required this.tripAdvicesData,
+    required this.maneuverData,
+    required this.evaluationData,
+    required this.metaData,
+    required this.transportationMode,
+    required this.declaredTransportationMode,
+    required this.unscored,
+    required this.calls,
+    required this.speedLimitContexts,
+  });
+
+  /// Creates a PigeonTrip instance
+
+  /// The itinerary ID
+  final String? itinId;
+
+  /// The start date
+  final String? startDate;
+
+  /// The end date
+  final String? endDate;
+
+  /// The departure city
+  final String? departureCity;
+
+  /// The arrival city
+  final String? arrivalCity;
+
+  /// The departure address
+  final String? departureAddress;
+
+  /// The arrival address
+  final String? arrivalAddress;
+
+  /// The itinerary ID
```

Comment:
"The **vehicle** ID"

Reply:
bien vu, corrigé sur 168fba8e9a76cb16947529e299e4b0a367753157